src/System.Security.Cryptography.Xml/src/project.json

```diff
@@ -0,0 +1,14 @@
+{
+  "frameworks": {
+    "netstandard1.7": {
+      "dependencies": {
+        "NetStandard.Library2": "2.0.0-beta-24814-01"
+      }
+    },
+    "net463": {
```

Comment:
We should make this net462 (and reference the 4.6.2 TargetingPack).

For the impl we could *probably* get away with net46, since only some consts changed in the API surface between 4.6 and 4.6.2.  But 4.6.2 is safer.